### DISCUSSION TITLE
Github Issue content used to be publicly accessible now new url links are private

### DISCUSSION THREAD

**Discussion by arrowmccross** [community/community#102851] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I have a public repository that I use the issues to upload images from my computer in order to generate a link that I can then embed on Quicksight. This was working fine for me up until recently, all the my previous images are still working and have a url as per https://user-images.githubusercontent.com/............ But any new issues I create now generate a link as per https://private-user-images.githubusercontent.com/............ I have not changed any of my settings, the repository is public. Any advice would be appreciated.

**Comment by Vijaykv5** [comment]
@arrowmccross  
I think it seems like there might be an issue with the repository's image hosting settings. 
GitHub typically generates image URLs with "user-images" for public repositories. The shift to `private-user-images` suggests a possible change in privacy settings. 
Check Once again your repository settings to ensure it's still set to public. If the problem persists, I don't think we have a straight forward solution please consider reaching out to GitHub support for assistance.

**Reply by arrowmccross** [reply]
Thanks sir for your, its a strange one to me as in the settings it still is defined as public. I will need to ask for further support I think. Thanks again

**Reply by Vijaykv5** [reply]
Yeah Just reach out to Github support!!
That would be the great option.

If found this useful I would recommend you to close this discussion(by marking this as an answer)
